outputs.tf

```diff
@@ -55,10 +55,10 @@ output "ebs_ids" {
 
 output "primary_network_interface_id" {
   description = "ID of the instance's primary network interface"
-  value       = "${aws_instance.default.primary_network_interface_id}"
+  value       = "${aws_instance.default.*.primary_network_interface_id}"
```

Comment:
since `${aws_instance.default.*.primary_network_interface_id}` returns a list,
it should be `${join("", aws_instance.default.*.primary_network_interface_id)}`
to return one item from the list

Reply:
same as above

Reply:
the output returns one ID, not a list (as the description says)